changelog.md

```diff
@@ -3,7 +3,8 @@
 ## Version 4
 
 * Split repository from [Corda](https://github.com/corda/corda) repository.
+* Update publish-utils so that setting its name property no longer triggers confguration. This is now applied after the project has been evaluated.
```

Comment:
Sorry to keep adding work but can you put this under a sub-version "4.0.<x>" (which is whatever snapshot you're currently on)